docs/flow/reference/components/discovery.relabel.md

```diff
@@ -1,31 +1,31 @@
 ---
 aliases:
-- /docs/agent/latest/flow/reference/components/targets.mutate
-title: targets.mutate
+- /docs/agent/latest/flow/reference/components/discovery.relabel
+title: discovery.relabel
 ---
 
-# targets.mutate
+# discovery.relabel
 
-`targets.mutate` rewrites the label set of the input targets by applying one or
-more `relabel_config` steps. If no relabeling steps are defined, then the input
-targets will be exported as-is.
+`discovery.relabel` rewrites the label set of the input targets by applying one
+or more `relabel_config` steps. If no relabeling steps are defined, then the
+input targets will be exported as-is.
 
-The most common use of `targets.mutate` is to filter Prometheus targets or
+The most common use of `discovery.relabel` is to filter Prometheus targets or
 standardize the label set that will be passed to a downstream component. The
 `relabel_config` blocks will be applied to the label set of each target in
 order of their appearance in the configuration file.
 
-Multiple `targets.mutate` components can be specified by giving them different
-labels.
+Multiple `discovery.relabel` components can be specified by giving them
+different labels.
 
 ## Example
 
 ```river
-targets.mutate "keep-backend-only" {
+discovery.relabel "keep_backend_only" {
```

Comment:
Block labels must now be valid identifiers. Personal reminder to run any examples present as we're cleaning up the components to verify everything's working properly.